workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
Let's go with 'tags' for now, and file an issue to rename everything to 
'metadata' (might be a good first task for Chris).  I'm over-thinking 
this...